docs/user-guide/persistent-volumes/index.md

```diff
@@ -151,6 +152,10 @@ Generally, a PV will have a specific storage capacity.  This is set using the PV
 
 Currently, storage size is the only resource that can be set or requested.  Future attributes may include IOPS, throughput, etc.
 
+### Volume Mode
+
+Prior to v1.9, the default behavior for all volume plugins was to create a filesystem on the persistent volume. With v1.9, the user can specify a volumeMode which will now support raw block devices in addition to fileystems. This feature is alpha in v1.9 and may change in the future.
+
```

Comment:
Can you add support matrix like this?
Also please mention available values for volumeMode. unspecified/Filesystem/Block

| Volume Plugin        | Filesystem| Block    |
| :---                 |   :---:   | :---:    |
| AWSElasticBlockStore | &#x2713;  | -        |
| AzureFile            | &#x2713;  | -        |
| AzureDisk            | &#x2713;  | -        |
| CephFS               | &#x2713;  | -        |
| Cinder               | &#x2713;  | -        |
| FC                   | &#x2713;  | &#x2713; |
| FlexVolume           | &#x2713;  | -        |
| Flocker              | &#x2713;  | -        |
| GCEPersistentDisk    | &#x2713;  | -        |
| Glusterfs            | &#x2713;  | -        |
| HostPath             | &#x2713;  | -        |
| iSCSI                | &#x2713;  | -        |
| PhotonPersistentDisk | &#x2713;  | -        |
| Quobyte              | &#x2713;  | -        |
| NFS                  | &#x2713;  | -        |
| RBD                  | &#x2713;  | -        |
| VsphereVolume        | &#x2713;  | -        |
| PortworxVolume       | &#x2713;  | -        |
| ScaleIO              | &#x2713;  | -        |
| StorageOS            | &#x2713;  | -        |

Reply:
Can we add examples for both Block and Filesystem?
May be we can add example for FC in volumes.md.

Reply:
@mtanino I am going to add this matrix to the other doc that Michelle pointed out